app/models/trust_level_setting.rb

```diff
@@ -8,7 +8,8 @@ def self.valid_value?(val)
   end
 
   def self.values
-    @values ||= valid_values.map { |x| { name: x.to_s, value: x } }
+    levels = TrustLevel.all
+    @values ||= valid_values.map { |x| { name: "#{x}: #{levels[x.to_i].name}", value: x } }
```

Comment:
Is `.name` the translated name? If not, we should use the translated version instead.

Reply:
It is already translated in https://github.com/discourse/discourse/blob/trust-level-settings/lib/trust_level.rb#L41